client/src/components/reader/Resource/Marker.js

```diff
@@ -51,8 +51,8 @@ class ResourceMarker extends Component {
               onMouseLeave={() => { this.setActiveAnnotation(null); }}
             >
               <i
-                className="manicon manicon-cube-fill"
-              ></i>
+                className="manicon resource-cube-fill"
+              >&nbsp;&nbsp;&nbsp;&nbsp;</i>
```

Comment:
This seems weird. Why do we have 4 non-breaking spaces here? Is this the best solution?